package.json

```diff
@@ -15,7 +15,8 @@
     "build-docs": "typedoc",
     "gh-pages": "gh-pages",
     "netlify:preview": "netlify deploy --site $NETLIFY_SITE_ID --auth $NETLIFY_AUTH_TOKEN --json --alias=$short_sha --message 'Deployment for $GITHUB_REF'",
-    "netlify:publish": "netlify deploy --site $NETLIFY_SITE_ID --auth $NETLIFY_AUTH_TOKEN --json --prod --message 'Deployment for $GITHUB_REF'"
+    "netlify:publish": "netlify deploy --site $NETLIFY_SITE_ID --auth $NETLIFY_AUTH_TOKEN --json --prod --message 'Deployment for $GITHUB_REF'",
+    "prepare": ""
```

Comment:
Is the prepare script needed if it is empty?

Reply:
it's needed so it doesn't get run in the CI, I was following this part of the docs https://typicode.github.io/husky/#/?id=disable-husky-in-cidockerprod

Reply:
got it, thanks!